test/experiment/ab_test.rb

```diff
@@ -258,6 +258,17 @@ def test_returns_the_same_alternative_consistently_when_using_probabilities
     end
   end
 
+  def test_uses_configured_probabilities_for_new_assignments
+    new_ab_test :foobar do
+      alternatives "foo" => 30, "bar" => 70
```

Comment:
Should we add a test to clarify behavior when the given probabilities don't add up to 100?